### DISCUSSION TITLE
'Merging' devices with Google Cast integration devices

### DISCUSSION THREAD

**Discussion by JeffersonBledsoe** [leikoilja/ha-google-home#358] [General]
Currently, the timer/ alarm devices that this integration creates appear separate to the devices containing the media_player entities created by the Google Cast integration. Just wondering if it would be possible for these devices to somehow be 'merged' so they appear as a single device in HA?

**Comment by KapJI** [comment]
Devices are grouped by integration, I don't think we should group entities from multiple integrations.

**Reply by leikoilja** [reply]
Unless we decide to merge these two integrations into one I agree with @KapJI, I don't think we can/should tamper with the entities created by Google Cast integration. 
If i m not mistaken, Google cast is a part of the HA 'core', which doesn't make it easier to find common grounds. Maybe as HA matures and start splitting components into separate projects with corresponding repos then we can get back to this question and considering merging the efforts? :)
